FidelLimited/rn-sdk
# Use bitrise to release the rn sdk

1. Added Bitrise continuous integration workflow
2. Updated React Native version of the example application
3. Updated build tools, minSdkVersion, targetSdkVersion to support latest RN
4. Updated Android bridge library unit testing dependencies
5. Added Jacoco code coverage report gradle tasks

Couldn't do for publishing the library more than we already have set up for the RN SDK (please check the task's description).

**How to test**
Check the latest Bitrise build and its logs. All steps should be succesful.
Check that the code coverage was uploaded for the latest commit of this branch. later, after merging this PR, the `develop` branch will also have a code coverage report, so it will display it in Codecov.
Check that the Android & iOS example apps work fine (please use the README instructions to install & test the example RN apps).
